scm/doc/TechGuide/chap_function.rst

```diff
@@ -0,0 +1,157 @@
+.. _`chapter: algorithm`:
+
+Algorithm
+=========
+
+Algorithm Overview
+------------------
+
+Like most SCMs, the algorithm for the CCPP SCM is quite simple. In a
+nutshell, the SCM code performs the following:
+
+-  Read in an initial profile and the forcing data.
+
+-  Create a vertical grid and interpolate the initial profile and
+   forcing data to it.
+
+-  Initialize the physics suite.
+
+-  Perform the time integration, applying forcing and calling the
+   physics suite each time step.
+
+-  Output the state and physics data.
+
+In this chapter, it will briefly be described how each of these tasks is
+performed.
+
+Reading input
+-------------
+
+The following steps are performed at the beginning of program execution:
+
+#. Call ``get_config_nml()`` in the ``scm_input`` module to read in the ``case_config`` and
```

Comment:
case_config was linked in the original version to the case_config sec

Reply:
Unfortunately one of the limits of Sphinx is that it is unable to combine links with code formatting. I have updated this to add a link as a compromise.